<launch>
   
  <arg name="map_file" default="/media/edo/My\ Passport/husky\ nav\ test/Daniele/250419-husky-apollo-complex-map/my_map.yaml"/>
  <node name="map_server" pkg="map_server" type="map_server" args="$(arg map_file)" />

  <arg name="bag_file" default="/media/edo/My\ Passport/husky\ nav\ test/Daniele/250419-husky-apollo-complex-map/complex-map_2019-04-25-17-00-51.bag"/>
  <node name="rosbag" pkg="rosbag" type="play" args="$(arg bag_file)" />
  
  
  <node name="load_wp" pkg="evalu_navi" type="load_wp.py" output="screen" >
    <rosparam command="load" file="$(find evalu_navi)/config/conf.yaml"/>
  </node>

</launch>
